src/pages/workspace/WorkspaceInitialPage.tsx

```diff
@@ -161,6 +161,12 @@ function WorkspaceInitialPage({policyDraft, policy: policyProp, policyMembers, r
             action: singleExecution(waitForNavigate(() => Navigation.navigate(ROUTES.WORKSPACE_TAGS.getRoute(policyID)))),
             routeName: SCREENS.WORKSPACE.TAGS,
         },
+        {
+            translationKey: 'workspace.common.taxes',
+            icon: Expensicons.Tax,
+            action: singleExecution(waitForNavigate(() => Navigation.navigate(ROUTES.WORKSPACE_TAXES.getRoute(policyID)))),
+            routeName: SCREENS.WORKSPACE.TAXES,
+        },
```

Comment:
Shouldn't it be displayed conditionally? 

I think, to see this option, the user has to enable this feature in the `More features` screen.

Reply:
Yes, but until `More Features` page is not created we have no condition to display or show it, I will add TODO comment for creators of `More Features` page